openstf/tasks/calculate_kpi.py

```diff
@@ -165,7 +165,9 @@ def calc_kpi_for_specific_pid(pid, start_time=None, end_time=None):
 
     # Raise exception in case of constant load
     if combined.load.nunique() == 1:
-        raise ValueError("The load is constant!")
+        structlog.get_logger().warning(
```

Comment:
I would suggest to always give the logger a name. I guess `structlog.get_logger(__name__)` should work well here.